assets/js/script.js

```diff
@@ -76,4 +76,9 @@ carousel.addEventListener('mouseleave', () => {
 
 carousel.addEventListener('mouseenter', () => {
   carousel.classList.add('hover');
-});
\ No newline at end of file
+});
+
+function openInNewTab(url) {
+  var newTab = window.open(url, '_blank');
+  newTab.focus();
+}
```

Comment:
Can't we simply use a `<a>` tag instead of this function?

Reply:
yes.

Reply:
Changes done.